src/store/modules/app/actions.js

```diff
@@ -162,6 +158,9 @@ const actions = {
     commit('setIso', iso);
     const keyboardLayout = iso ? 'enableIso' : 'disableIso';
     this.commit(`keycodes/${keyboardLayout}`);
+    this.commit('keymap/updateKeycodeNames');
```

Comment:
new hooks go here for now. They basically force keymap to update all the names, for both the keymap and the tester.